client/components/ProductList.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ProductListItem from './product-list-item';
+
+export default class ProductList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      products: []
+    };
+  }
+
+  componentDidMount() {
+    this.getProducts();
+  }
+
+  getProducts() {
+    fetch('/api/products')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ products: data });
+      })
+      .catch(err => {
+        this.setState({
+          products: [`the error is ${err}`]
```

Comment:
What renders if there's an error and `this.state.products` is an array with just a string? For now, just `console.error()` the `err`